lib/housekeeping.js

```diff
@@ -58,53 +60,66 @@ class HouseKeeper {
     //       }
     //     },
     //     ...
-    //   ]
+    //   ],
+    //   ${region} : [
+    //   ...
+    //   ],
+    //   ...
     // }
 
     //assert(typeof totalsByCategory === 'object');
 
-    //assert(Object.keys(totalsByCategory).length === 1);
-    let region =  Object.keys(totalsByCategory)[0];
+    // Keep track of overall totals, over all regions
+    let totalGbUsage = 0;
+    let totalCount = 0;
+
+    Object.keys(totalsByCategory).forEach(function(region) {
     
-    let volumeTypes = totalsByCategory[region];
-    //assert(typeof volumeTypes === 'object');
-    //assert(typeof volumeTypes.hasOwnProperty('length'));
+      let volumeTypes = totalsByCategory[region];
+      //assert(typeof volumeTypes === 'object');
+      //assert(typeof volumeTypes.hasOwnProperty('length'));
 
-    volumeTypes.forEach(function(data) {
-      //assert(typeof data === 'object');
+      volumeTypes.forEach(function(data) {
```

Comment:
I think `volumeTypes` is an object. If it is, we're going to have to traverse through by keys:
`Object.keys(volumeTypes).foreach(function(type) {}`

I made a possible implementation in https://github.com/rstiyer/ec2-manager/pull/3 with some changes to `active` and `unused`.

Reply:
If you look at my assumed format, `volumeTypes` is in fact a list of objects where the key of each object is the type and the value are the GB and count totals among volumes of that type in the given region.

Reply:
I don't think there is any difference at all, to be honest. I just went with this approach because we don't know exactly how many different volume types we can expect and it's more natural to loop over a list of objects rather than the keys of the parent object.

Reply:
I'm sorry I was getting confused between the brackets and didn't realize that it was a list of objects. Your implementation makes sense then. I agree that it is more natural to loop over a list of objects. However, I'm not a huge fan of how we would have to access the elements: e.g. `let type = Object.keys(data)[0];` is not that readable compared to just getting the `type` as the key in the `.foreach()` loop.

Reply:
I think your implementation is probably better! :) Definitely clearer, so let's go for it!

Reply:
Awesome. How about we merge your PR into `calculate-and-update-totals`? We can merge my PR with this new implementation in afterwards. We're still not merging it into the canon repo so it's OK that @imbstack and @djmitche hasn't had a chance to look at this PR yet.

Or we could also just copy/paste my changes to `_submitVolumeTotals()` into your PR then merge it in later.

Reply:
Hopefully, the PR for merging `calculate-and-update-totals` into the canon repo wouldn't be too big.